db/schema.yml

```diff
@@ -1232,6 +1232,7 @@ generic_objects:
 - generic_object_definition_id
 - created_at
 - updated_at
+- properties
```

Comment:
Yeah, this looks fine to me, this file will get changed with all schema migrations now. As long as the changes here match up with what you were trying to migration then we should be good.

I use this to check that the schemas are the same on both sides of a replication connection, sorry for the confusion.

@gmcculloug We also have a `schema.rb` which is a rails thing, but isn't as nice to parse and may not have been as careful with column ordering.